src/shared/internal-calendar/internal-calendar.tsx

```diff
@@ -118,6 +120,9 @@ export const Component = (
             case "week":
                 isDisabled = !selectedStartDate && !selectedEndDate;
                 break;
+            case "fixed-range":
+                isDisabled = !selectedStartDate && !selectedEndDate;
```

Comment:
can this be combined with `case "range"` instead? to allow for manual clearing of the form input